tasks/main.yml

```diff
@@ -184,7 +184,7 @@
         interface: "{{ os_images_interface | default(omit, true) }}"
         name: "{{ item.name ~ '-kernel' }}"
         state: present
-        is_public: yes
+        is_public: "{{ os_images_public }}"
```

Comment:
Could you add a bool filter to allow yes/no etc?

`"{{ os_images_public | bool }}"`

Reply:
Done